feeadjuster/feeadjuster.py

```diff
@@ -129,8 +135,16 @@ def init(options: dict, configuration: dict, plugin: Plugin, **kwargs):
 plugin.add_option(
     "feeadjuster-threshold",
     "0.05",
-    "Channel balance update threshold at which to trigger an update. Note "
-    "it's fuzzed.",
+    "Channel balance update threshold at which to trigger an update. "
+    "Note: it's also fuzzed by 1.5%",
+    "string"
+)
+plugin.add_option(
+    "feeadjuster-imbalance",
+    "50",
+    "Percentage at which channel imbalance the feeadjuster should start acting. "
+    "Default: 50 (always). Set higher or lower values to limit feeadjuster's "
+    "activity and increase your privacy. E.g. 30 for '70/30' or 60 for '40/60'.",
```

Comment:
Oh, sorry wasn't explicit: by "float" i meant 0.05 instead of 50

Reply:
you mean "0.5" ?

Reply:
Can do that...

Reply:
done. Also added a small addition to the test to check this variable has been processed correctly.